kolibri/plugins/device/assets/src/views/ManageTasksPage/TaskPanel.vue

```diff
@@ -130,6 +130,11 @@
 
   export default {
     name: 'TaskPanel',
+    metaInfo() {
+      return {
+        title: `${this.formattedPercentage} - ${this.dynamicTitle}`,
```

Comment:
If `this.formattedPercentage` is equal to `''` then we probably don't want to include it in the title at all, right?

So this would be more like:

```
this.formattedPercentage === '' ? this.dynamicTitle : `${this.formattedPercentage} - ${this.dynamicTitle}`